src/app/groups/containers/group-log-view/group-log-view.component.html

```diff
@@ -55,12 +55,19 @@
                 <ng-template #iconSection>
                   <i class="ph-duotone item-icon" [ngClass]="{
                     'ph-flag-checkered': rowData.activityType === 'result_started',
-                    'ph-floppy-disk-back': rowData.activityType === 'saved_answer'
+                    'ph-floppy-disk-back': rowData.activityType === 'saved_answer' || rowData.activityType === 'current_answer'
                   }"></i>
                 </ng-template>
               </div>
               <div class="item-title">
                 <p class="item-title-caption">{{ rowData.activityType | logActionDisplay }}</p>
+                <a
+                  class="alg-link"
+                  [routerLink]="rowData.item | rawItemRoute | withAnswer: { id: rowData.answerId } | url"
+                  *ngIf="!groupId && rowData.activityType === 'saved_answer' && rowData.answerId"
```

Comment:
The fact that groupId not defined means "myself" is not well documented.. I had to check the doc to understand. Can you document the component parameter? 
I think that it is better if all row containing current-user's own answer can be viewed... even when I am listing the progresses of a group I am member of.... so you should base yourself on row participant instead. (or in addition)

Reply:
Fixed

Reply:
Fixed. I've add new test case

Reply:
Fixed